cast/src/lib.rs

```diff
@@ -10,9 +9,18 @@ use starknet::{
     providers::jsonrpc::{HttpTransport, JsonRpcClient},
     signers::{LocalWallet, SigningKey},
 };
+use std::collections::HashMap;
 use std::fs;
 use url::Url;
 
+// Taken from starknet-rs
+pub const UDC_ADDRESS: FieldElement = FieldElement::from_mont([
```

Comment:
we could also take it from starkscan/starkware docs in hex representation and use it here, right? I guess it could be easier to track